tests/mbed/CMakeLists.txt

```diff
@@ -17,17 +17,16 @@ if (NOT "${build_type}" STREQUAL "DEBUG")
 	set(is_nondebug_build TRUE)
 endif()
 foreach(testcase ${alltests})
-	string(FIND "${testcase}" "selftest" found_selftest)
-	if(${found_selftest} GREATER 0 )
-		get_testcase_name(${testcase} name "../../3rdparty/mbedtls/mbedtls/programs/test/")
-		add_enclave_test(tests/mbed/${name} libmbedtest_host libmbedtest-${name}_enc)
+        string(COMPARE EQUAL "${testcase}" "selftest" found_selftest)
```

Comment:
There's an extra space here at the start

Reply:
Ah, the original file has tabs. I'm not bothering to change it for now.